vault-plugins/shared/jwt/jwks_model.go

```diff
@@ -0,0 +1,43 @@
+package jwt
+
+import (
+	"crypto/ed25519"
+
+	"github.com/hashicorp/go-memdb"
+)
+
+const (
+	JWKSType = "jwks" // also, memdb schema name
+)
+
+func JWKSSchema() *memdb.DBSchema {
+	return &memdb.DBSchema{
+		Tables: map[string]*memdb.TableSchema{
+			JWKSType: {
+				Name: JWKSType,
+				Indexes: map[string]*memdb.IndexSchema{
+					"id": {
+						Name:   "id",
+						Unique: true,
+						Indexer: &memdb.UUIDFieldIndex{
+							Field: "UUID",
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+type JWKS struct {
+	UUID   string `json:"uuid"`
+	Pubkey *ed25519.PublicKey
```

Comment:
pudkey will not marshal and not send in replicas. Do I right?

Reply:
Nope :) It will